authomatic/providers/__init__.py

```diff
@@ -382,14 +383,26 @@ def _fetch(self, url, method='GET', params=None, headers=None, body='', max_redi
         self._log(logging.DEBUG, ' \u251C\u2500 params: {0}'.format(params))
         self._log(logging.DEBUG, ' \u2514\u2500 headers: {0}'.format(headers))
         
+        http_proxy = os.environ.get('http_proxy')
+        https_proxy= os.environ.get('https_proxy')
+        type,https_host=https_proxy.split('//')
```

Comment:
``type`` is a built_in, to avoid conflicts in names use ``type_`` as it's done often in Python.
Also add a space after the ``,`` (pep8)